bot/settings.py

```diff
@@ -226,6 +226,7 @@ def inline_add_repo(update, context):
     results = []
     if access_token:
         filtered_repositories = []
+        registered_repos = context.chat_data.setdefault('repos', {})
```

Comment:
Inline callbacks have no chat_data as they don't receive the chat.
This leads me to believe that this might be quite difficult to implement.